fixtures/mypkg/setup.py

```diff
@@ -0,0 +1,10 @@
+"""A simple setup file for this test package."""
+
+from setuptools import setup
+
+setup(
+    name="pypiserver_mypkg",
+    description="Test pkg",
+    version="1.0.0",
+    packages=["mypkg"],
+)
```

Comment:
what about just using the `centodeps` approach from the unittests to create a simple package?

Reply:
Personally I like having it checked in, because it more clearly mirrors the "real world," and would also allow us to eventually add fixture packages that do things like include C extensions or utilize alternate build systems (like poetry) more easily. It's largely six of one and half dozen of the other, but I think it's nice when experimenting to be able to go look at the file, run `setup.py` on it directly, upload it to a locally running pypiserver, etc.